src/sourmash/commands.py

```diff
@@ -1242,14 +1242,24 @@ def prefetch(args):
     if args.save_matching_hashes:
         filename = args.save_matching_hashes
         notify(f"saving {len(ident_mh)} matched hashes to '{filename}'")
-        ss = sig.SourmashSignature(ident_mh)
+
+        sig_name = ''
+        if query.name:
+            sig_name = f"{query.name}-known"
+
+        ss = sig.SourmashSignature(ident_mh, name=sig_name)
         with open(filename, "wt") as fp:
             sig.save_signatures([ss], fp)
 
     if args.save_unmatched_hashes:
         filename = args.save_unmatched_hashes
+
+        sig_name = ''
+        if query.name:
+            sig_name = f"{query.name}-known"
```

Comment:
Shouldn't this be `-unknown`? due to the `unmatched_hashes`.

Reply:
yep. good catch!

Reply:
fixed in a6801d5f57e5a4c